.github/workflows/build.yml

```diff
@@ -268,6 +268,12 @@ jobs:
         if: contains(github.ref_name, 'main')
         run: |
           yarn test:visual:ci
+      - uses: actions/upload-artifact@v3
+        if: always()
+        with:
+          name: visual-report
+          path: visual-report/
+          retention-days: 3
```

Comment:
Does this mean that Friday builds will only last until Monday? I think we could up this to a week unless there are any other issues

Reply:
Fair point. Will update to 7.